app/models/employee.rb

```diff
@@ -168,7 +163,7 @@ def emp_no
     date = Date.today.strftime('%Y%m%d')
     self.employee_number = date.to_s + '1' if Employee.first.nil?
     self.employee_number = date.to_s + \
-      Employee.last.id.next.to_s unless Employee.first.nil?
+                           Employee.last.id.next.to_s unless Employee.first.nil?
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines. so thats why there is given space.